lib/message.rb

```diff
@@ -0,0 +1,26 @@
+require 'twilio-ruby'
+
+class Message
+  # Note that I'm using test credentials for sid, auth and from_no
+  # As far as I can tell, there's no way of using live SMS without paying
+
+  def initialize
+    @account_sid = ENV['TWILIO_ACCOUNT_SID']
+    @auth_token = ENV['TWILIO_AUTH_TOKEN']
+    @sender = ENV['MY_FROM_NUMBER']
+  end
+
+  def send(customer, time)
+    message = "Thank you! Your order was placed and will be delivered before #{time+(60*60)}"
+    # set up a client to talk to the Twilio REST API
+    @client = Twilio::REST::Client.new @account_sid, @auth_token
+
+    @client.account.messages.create({
+      :from => @sender,
+    	:to => customer.mobile_number,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Tab detected.